packages/integrations/useAxios/demo.vue

```diff
@@ -4,6 +4,12 @@ import { useAxios } from '.'
 
 const { data, isLoading, isFinished, execute, abort, isAborted } = useAxios(
   'https://jsonplaceholder.typicode.com/todos/1',
+  {},
```

Comment:
Why the mutating for `useAxios`

Reply:
This is caused by merging other master branches. That is the effect of one of my previous commits on useAxios

Reply:
When I submitted it, he reminded me that there were updates to be merged. After automatic merging, useAxios changed.

Reply:
Could you revert the changes in `useAxios`, just delete the additional lines.
BTW, tests are failed, you can test them on your local before you commit by run `pnpm run test:all`.

Reply:
I restored useAxios and fixed the tests

Reply:
Tests still failed, you can run `pnpm run test:all` to determine if the tests are pass

Reply:
I re-opened a pr because I didn't know how to clean up useless commit records, and the reason I was testing a lot of errors was because I imported cloneFnJSON from @vueuse/core.